internal/db/rdb/mysql/mysql_test.go

```diff
@@ -1260,245 +1332,1533 @@ func Test_mySQLClient_SetMeta(t *testing.T) {
+			return test{
+				name: "return error when meta vector is invalid",
+				args: args{
+					ctx: ctx,
+					mv:  m,
+				},
+				fields: fields{
+					session: &mock.MockSession{
+						BeginFunc: func() (dbr.Tx, error) {
+							tx := new(mock.MockTx)
+							tx.RollbackFunc = func() error {
+								return nil
+							}
+							tx.RollbackUnlessCommittedFunc = func() {}
+							return tx, nil
+						},
+					},
+					connected: func() (v atomic.Value) {
+						v.Store(true)
+						return
+					}(),
+				},
+				want: want{
+					err: errors.ErrRequiredMemberNotFilled("vector"),
+				},
+				afterFunc: func(args) {
+					cancel()
+				},
+			}
+		}(),
+		func() test {
+			ctx, cancel := context.WithCancel(context.Background())
+			m := new(metaVector)
+			m.meta.Vector = []byte("0.1,0.2,0.9")
+			err := errors.New("insertbysql ExecContext error")
+			return test{
+				name: "return error when insertbysql ExecContext returns error",
+				args: args{
+					ctx: ctx,
+					mv:  m,
+				},
+				fields: fields{
+					session: &mock.MockSession{
+						BeginFunc: func() (dbr.Tx, error) {
+							tx := new(mock.MockTx)
+							tx.RollbackFunc = func() error {
+								return nil
+							}
+							tx.RollbackUnlessCommittedFunc = func() {}
+							tx.InsertBySqlFunc = func(query string, value ...interface{}) dbr.InsertStmt {
+								s := new(mock.MockInsert)
+								s.ColumnsFunc = func(colum ...string) dbr.InsertStmt {
+									return s
+								}
+								s.ExecContextFunc = func(ctx context.Context) (sql.Result, error) {
+									return nil, err
+								}
+								s.RecordFunc = func(structValue interface{}) dbr.InsertStmt {
+									return s
+								}
+								return s
+							}
+							return tx, nil
+						},
+					},
+					connected: func() (v atomic.Value) {
+						v.Store(true)
+						return
+					}(),
+				},
+				want: want{
+					err: err,
+				},
+				afterFunc: func(args) {
+					cancel()
+				},
+			}
+		}(),
+		func() test {
+			ctx, cancel := context.WithCancel(context.Background())
+			m := new(metaVector)
+			m.meta.Vector = []byte("0.1,0.2,0.9")
+			err := errors.New("loadcontext error")
+			return test{
+				name: "return error when select loadcontext returns error",
+				args: args{
+					ctx: ctx,
+					mv:  m,
+				},
+				fields: fields{
+					session: &mock.MockSession{
+						BeginFunc: func() (dbr.Tx, error) {
+							tx := new(mock.MockTx)
+							tx.RollbackFunc = func() error {
+								return nil
+							}
+							tx.RollbackUnlessCommittedFunc = func() {}
+							tx.InsertBySqlFunc = func(query string, value ...interface{}) dbr.InsertStmt {
+								s := new(mock.MockInsert)
+								s.ColumnsFunc = func(colum ...string) dbr.InsertStmt {
+									return s
+								}
+								s.ExecContextFunc = func(ctx context.Context) (sql.Result, error) {
+									return nil, nil
+								}
+								s.RecordFunc = func(structValue interface{}) dbr.InsertStmt {
+									return s
+								}
+								return s
```

Comment:
I think the folwloing patter is more simple. :thinking:
```suggestion
return mock.MockInsert {
	ExecContextFunc: func(ctx context.Context) {									
}
```

Reply:
How I return itself by this coding?
```go
return mock.MockInsert{
  FromFunc: func(table string) dbr.InsertStmt {
    // this will fail due to invalid memory address or nil pointer dereference
    return new(mock.MockInsert)
  }
}
```

Reply:
sorry. I made a mistake:bow:
your opinion is correct! thank you

Reply:
if I have to only `ExecContextFunc`, your suggestion is good 👍